docs/source/quickstart/index.rst

```diff
@@ -266,3 +266,13 @@ Build and Quality Status
 .. image:: https://readthedocs.org/projects/avocado-framework/badge/?version=latest
    :target: https://avocado-framework.readthedocs.io/en/latest/
    :alt: Documentation Status
+
+
+.. _Avocado User's Guide: https://avocado-framework.readthedocs.io/en/latest/guides/user/index.html
+.. _Installing section on Avocado User’s Guide: https://avocado-framework.readthedocs.io/en/latest/guides/user/chapters/installing.html#installing
+.. _Test types: https://avocado-framework.readthedocs.io/en/latest/guides/user/chapters/concepts.html#test-types
```

Comment:
The `https` links are not ideal, because the destination is absolute and not relative.  Suppose the user is reading documentation on version 89.0, and clicks on this link: he is redirected to the "latest" version which may be different (or may not even exist any longer).  It's better to use sphinx ``ref``s.

Reply:
@clebergnu We're also using this file as README.rst that's shown when people browse https://github.com/avocado-framework/avocado . Will the sphinx's ref work properly there?

Reply:
If I recall correctly, the `README.rst` is the only exception because we link it to the main GitHub page.

Reply:
> @clebergnu We're also using this file as README.rst that's shown when people browse https://github.com/avocado-framework/avocado . Will the sphinx's ref work properly there?

Oh, I missed that, and no, it won't.  So it looks like we're between the less of two evils, and I think your approach is the least of them.